tests/test_wave_01.py

```diff
@@ -142,13 +143,15 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
+    assert updated_data["watched"][1]["title"] == MOVIE_TITLE_1
```

Comment:
This works well but you can also test to see if the movie is in the watched list as well as some other assertions to ensure no unwanted list modifications have occurred:
```python
    assert movie_to_watch in updated_data["watched"]
    assert FANTASY_2 in updated_data["watched"]
    assert FANTASY_1 in updated_data["watchlist"]
    assert movie_to_watch not in updated_data["watchlist"]
```